lib/chef/knife/ec_restore.rb

```diff
@@ -281,6 +278,8 @@ def upload_org_data(name)
           ['admins', 'billing-admins', 'public_key_read_access'].each do |group|
             restore_group(Chef::ChefFS::Config.new, group)
           end
+         rescue Net::HTTPServerException => ex
```

Comment:
I believe we need to break down the `upload_org_data` fun into small peaces so we can safely call `rescue` within those new small fun. Just as we do in `download_org_data` inside `ec_backup.rb`.